fffscrapy/spiders/katapult_ua.py

```diff
@@ -0,0 +1,51 @@
+import scrapy
+from scrapy.http import HtmlResponse
+import json
+
+
+class KatapultUaSpider(scrapy.Spider):
+    encoding = "utf-8"
+    name = 'katapult_ukraine'
+    base_url = 'https://katapult-magazin.de/de'
+    custom_settings = {
+        "FEED_EXPORT_ENCODING": encoding
+    }
+
+    def start_requests(self):
+        urls = [
+            self.base_url + '/artikel/liveblog-zur-ukraine'
+        ]
+        for url in urls:
+            yield scrapy.Request(url, callback=self.parse)
+
+    def parse(self, response, **kwargs):
+
+        # parse and get raw html between selector, then split based on date selector
+        previews = response.css('section.content.content-article').extract()[0] \
+            .split('<div class="column text-gray-dark">')
+
+        # remove first element (page intro)
+        del previews[0]
+
+        for preview in previews:
+            # append split div to have valid html again
+            preview = '<div class="column text-gray-dark">' + preview
+            # manually create parseable response
+            response = HtmlResponse(url="dummy", body=preview, encoding=self.encoding)
+            # extract data from dummy response using selectors
+            date = response.css('div.column.text-gray-dark::text').get(default='').strip()
+
+            # multiline texts may occur
+            content = '\n'.join(response.css('p::text').getall()).strip()
+
+            # multiple images may occur
+            images = response.css('figure.figure-image.center img')
+            image_links = []
+            for image in images:
+                image_links.append(image.attrib['data-srcset'].split(',')[-1].strip().split(' ')[0])
+
+            yield {
+                'date': date,
+                'content': content,
+                'images': json.dumps(image_links)
```

Comment:
The idea behind this is to have a list of images bc katapults sometimes releases multiple images with the same timestamp (see e.g. @ 01.03.2022 07:30 at https://katapult-magazin.de/de/artikel/liveblog-zur-ukraine). Therefore in order to avoid duplicate the same entry the intention is to have a list of images. This has two advantages: 1) less required space in the file and better readability, 2) when using this json in a downstream app, its on the developer to either create multiple entries with the same timestamp per image or use an aggregated version of multiple images as e.g. instagram does for multiple images. 

does this makes sense? 🤔

Reply:
Yes that's great! But as it is now, the `images` will be a string, ie. `images: "["path-to-img1", "path-to-img2"]"` and not a list of strings `images: ["path-to-img1", "path-to-img2"]`. Is this behaviour intended? Or the other way around: I don't think we need `json.dumps` here 🙂

Reply:
fair point - thanks for taking care of this - adapted :)